libs/blocks/z-pattern/z-pattern.js

```diff
@@ -86,5 +86,11 @@ export default function init(el) {
   const mediaItems = el.querySelectorAll(':scope > .media');
   mediaItems.forEach((i) => {
     initMedia(i, false);
```

Comment:
```suggestion
    variants.forEach((v) => {
      if (el.classList.contains(v)) i.classList.add(v);
    });
    initMedia(i, false);
```
With the above array, we can reduce some redundancy.